### ISSUE TITLE
winbtrfs stopped working, can't uninstall and reinstalling doesn't work

### ISSUE THREAD

**Issue by queenbiscuit311** [maharmstone/btrfs#659]
I'm using winbtrfs installed through chocolatey on a recent install of Windows 11 (23H2, build 22631.3527). It was working fine until it just sort of stopped working. None of my btrfs volumes are detected. Running the command in the readme to uninstall just does this:

![image](https://github.com/maharmstone/btrfs/assets/88851292/c7b27cc6-7079-4e27-a0be-4e3746af5ea5)

Uninstalling via chocolatey and reinstalling doesn't work either, neither does uninstalling via chocolatey and installing manually. I've tried checking system files and it said it found corrupt files but then nothing changed when i tried all of these steps again.

**Comment by queenbiscuit311** [comment]
Update: i tried manually uninstalling via modifying the registry key to disable btrfs and rebooting then uninstalling the registry keys and deleting btrfs.sys, then rebooting again, then reinstalling. Still no dice.

**Comment by queenbiscuit311** [comment]
The problem was: I found out how to get my linux install to work with secure boot, so i turned it on, and as the readme states that breaks the driver unless you set a registry key. I somehow didn't make the connection. i set the key, and it was fixed. I seem to have glossed over it when I looked at the readme because it specifically says that this is a windows 10 issue, and not being on windows 10 I ignored it. May be worth updating the readme to specify it also applies to windows 11, since specifically mentioning windows 10 and not 11 may imply that 11 isn't affected.
